### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by reeseovine** [fdw/rofimoji#67] [General]
Would it make sense to keep a roadmap/triage of features and fixes we want in Rofimoji? I like contributing to this repo but I want to make sure my priorities are not taking away from things you think are more important. With a roadmap, I (and others) could pick off things at the top to work on, and suggest things we want that could be implemented at a later time. Let me know what you think!

**Comment by fdw** [comment]
I'm really humbled by your (and other's) enthusiasm for this tiny project. Thank you! 😊

So far, `rofimoji` started with scratching my own itch and then iteratively adding features that either some users or I wanted or thought useful. I think this is quite a good mechanism to improve with actually useful features.
I also don't want `rofimoji` to become too bloated; it is and should be in essence a small utility to help you pick unusual characters easily. For example, I hesitated for quite some time to add external dependencies, and now I'm thinking of removing them again. Similarly, performance is a very important topic for me (as you noticed 😉). Thus, new features always must be weighted for their cost and benefit.

In summary, if you personally want a feature, feel free to implement it and open a PR, or just open an issue. I'm willing to look at anything, if it helps someone and if it satisfies my "smallness" requirements. But I don't have an agenda nor do I think something like this should have one.

**Reply by reeseovine** [reply]
Sounds good, thank you for the response. I've already forgotten what idea was bouncing around in my head, but that's okay.

**Reply by fdw** [reply]
Sorry, but I wanted to really think about this before answering 🙂
